Dockerfile

```diff
@@ -1,21 +1,17 @@
-FROM sonatype/nexus3:3.16.1
+FROM sonatype/nexus3:3.18.1
 MAINTAINER Dwolla Dev <dev+docker-nexus3-crowd@dwolla.com>
 LABEL org.label-schema.vcs-url="https://github.com/Dwolla/docker-nexus3-crowd"
 
 USER root
 COPY startup.sh /
-RUN yum install -y git which && \
-    curl -o /etc/yum.repos.d/epel-apache-maven.repo https://repos.fedorapeople.org/repos/dchen/apache-maven/epel-apache-maven.repo && \
-    yum install -y apache-maven && \
+RUN dnf install -y git which maven && \
```

Comment:
@charlesdunbar 's suggestion, using dnf for dependencies, and removing unneeded stuff about adding the EPEL